### DISCUSSION TITLE
Which Sequent Multi Relay Board

### DISCUSSION THREAD

**Discussion by subcritical71** [tagyoureit/nodejs-poolController#691] [REM] [answered]
Really basic question here, which 8 relay board from Sequent currently works with REM?

The Sequent website and Amazon have multiple names which are basically the same but different. It's quite confusing and I have already ordered one board which appears to not be compatible (Home Automation 8-Layer Stackable HAT for Raspberry Pi [Link](https://www.amazon.com/dp/B08GJ8MLNJ?psc=1&ref=ppx_yo2ov_dt_b_product_details) the board is labeled HOME-AUTOMATION-V3.0).

I'm thinking the one that is compatible is this one ([Link](https://www.amazon.com/Eight-RELAYS-8-Layer-Stackable-Raspberry/dp/B07KRKS67G?ref_=ast_sto_dp))

Thanks for any help.

**Comment by tagyoureit** [comment, marked as answer]
The [Home Automation](https://sequentmicrosystems.com/collections/home-automation/products/raspberry-pi-home-automation-card) board doesn't have all of the inputs needed for a Nixie (custom build).  The [Industrial Automation](https://sequentmicrosystems.com/collections/industrial-automation/products/industrial-raspberry-pi) or [Building Automation](https://sequentmicrosystems.com/collections/building-automation/products/building-automation-br-8-layer-stackable-hat-v4-br-for-raspberry-pi) cards do have the necessary inputs.

If you just need relays, then the [8-relay](https://www.amazon.com/Eight-RELAYS-8-Layer-Stackable-Raspberry/dp/B07KRKS67G?ref_=ast_sto_dp) is the right card.

What are you looking to do/build?

**Reply by subcritical71** [reply]
Thanks for the reply. I'm slowly getting my build together. It currently consists of two pumps, one on RS-485 (IntelliFlo VSF) the other on relays (SuperFlo VS), chlorinator (IC40). All these items are connected to rpi and working. The relays for the VS pump are currently on a relay board I had lying around, I will transfer those to the 8-relay board.

To go is a heater (wip - on the 485 bus but doesn't seem to be working), color changing lights (1 relay), filter pressure, air temp, and water(x2) temp sensors.

(Not too distant) future I want to have expansion capability for pH and ORP as well as an acid dosing system. Future will be valve automation for my water features. 

I do have the Bldg Automation board on the way and will order the 8-relay board you linked. I think that will cover most of the items I have listed. Thanks again.

**Reply by tagyoureit** [reply]
No no no!  Do NOT connect the pumps (or the chlorinator) directly to the regular Sequent Relays.  They aren't rated for that.  There is a special [high voltage Sequent relay](https://sequentmicrosystems.com/collections/all-io-cards/products/three-40a-240v-relays-rs485-for-raspberry-pi) but I don't think we've seen it in the wild yet.

Please read [this section](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#sequent-microsystems-8-relay-hat) of the Nixie Wiki *very* carefully.

**Reply by rstrouse** [reply]
@tagyoureit is correct regarding the main power for the pump, but if you are referring to the dry contact Speed Settings for the SuperFlo VS,  these are fine on the Relay 8.  However, do not connect an inductive load such as the main pump power or chlorinator transformer to any relay that is not rated as a contactor (3hp relay).  It will fail in an unhappy manner as the contacts break down.

**Reply by subcritical71** [reply]
Sorry, I misspoke on that one. I am using the relays through the pentair cable/dry contact system not the actual power lines. 

Good to know about the chlorinator. I've been wondering if I should leave the transformer powered up full time or cycle based on pool pump operation. The 'safe' way seems to be power through a relay that is only energized when the pool pump is running. That is NOT how it was wired from the previous owners installation. It was on the pool pump breaker but had no feedback other than the flow switch in the chlorinator. 

I actually have one of the 3 heavy duty relay modules from Sequent. I went through and tried to figure out how to add it to the list but that is beyond my skill level as it seems the communication with that module is different than the others.

**Reply by tagyoureit** [reply]
Thanks for the clarification.

The chlorinator should be wired to the [filter](https://github.com/tagyoureit/nodejs-poolController/wiki/Nixie-Controller-Setup#filters) circuit.  Whenever the main pump is on, the chlorinator should be energized (see [high current circuits](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#high-current-circuits)).  

For the HV relay, we can reverse engineer the board commands from the [Github repo](https://github.com/SequentMicrosystems/3relind-rpi).  Now we know someone has it out there, either @rstrouse or I can add it to the list of modules and you'll be able to test it out for us.

**Reply by subcritical71** [reply]
Sounds good. It would be nice to have this work with the pool controller.

**Reply by subcritical71** [reply]
Well, good news I think. After @rstrouse updated REM with the 4rel4in v4 card support last week I decided to give it a go with my 3relind v1.1 card.

The relays all work as expected, in the correct order, on is on and off is off. So making an entry for just the first 3 relays and removing the input portions should fix things up nicely. Or I could just live with the 4rel4in settings.

Also, REM automatically discovers the I2C port at address 96, (0x60) for this card.

**Reply by NWSAlpine** [reply]
I just bought all the boards I need from them but missed this high voltage relay. I would also need one for my aquarite since I have an intelliflo VSF pump that will be wired directly to the sub panel.

Did you wire your aquarite for 220v or 110v with this hat https://sequentmicrosystems.com/collections/all-io-cards/products/three-40a-240v-relays-rs485-for-raspberry-pi

**Reply by subcritical71** [reply]
I don't have an Aquarite, but do plan to use that relay for my Intellichem SWG using 110v as that is how it is currently wired. I'm not even sure if 220v is an option. The relay is rated for either scenario though.

**Reply by NWSAlpine** [reply]
I think I’m going to keep it 220 and just use a 3hp relay. It’s currently wired to an Intermatic digital dual timer 220 leftover from my old Hayward 2 speed. That’s getting ripped out but keeping the Intermatic load center. I ordered an Altilex din box for the Pi stack below:

Pi 4 4Gb -> sequent fan hat -> sequent building automation v4 -> sequent 8 relay -> sequent super watchdog ups. 

Just need to figure out how to wire my Pentair microbrites. I will have 5 that I want to switch as two separate zones for the main body and the bubbler lights separate.

**Reply by tagyoureit** [reply]
The microbrites are supposed to be compatible with the intellibrites, but it looks like the number of toggles for the color modes might be different.  When you get them installed, let us know if we need to create a separate microbrite light type.
